lib/google/longrunning/operations_client.rb

```diff
@@ -147,7 +148,7 @@ def initialize \
         google_api_client << " grpc/#{GRPC::VERSION}"
         google_api_client.freeze
 
-        headers = { :"x-goog-api-client" => google_api_client }
+        headers = { :"x-goog-api-client" => google_api_client }.merge(_headers)
```

Comment:
This can also be written like so:

```suggestion
        metadata ||= {}
        metadata[ :"x-goog-api-client"] ||= google_api_client
```